src/main/java/duke/DialogBox.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.scene.shape.Circle;
+
+/**
+ * A DialogBox handles the DialogBox in the GUI for both user and Duke
+ */
+public class DialogBox extends HBox {
+
+    private Label text;
+    private ImageView displayPicture;
+
+    /**
+     * Handles DialogBox in the GUI for both the user and Duke
+     * @param l label of the DialogBox
```

Comment:
Maybe a empty line between the description and @param would make the comment more similar to the coding standard for comments.